gulpfile.babel.js

```diff
@@ -27,7 +27,7 @@ const reload = browserSync.reload;
 var args = require("yargs").argv;
 // Server used for serving remote url"s
 // "http://136.231.10.175:8888";
-var apiServer = "http://delft3dgt-django:8000";
+var apiServer = "https://delft3dgt-nginx";
```

Comment:
This reverts the previous change, which was not supposed to be committed in the first place.

This change has no influence on the GUI in production.